From 5c62773cbda0041b3caee9b88ae03a40dbded983 Mon Sep 17 00:00:00 2001
From: Leandro Padilla <70078738+leanpadilla@users.noreply.github.com>
Date: 12-Nov-2020 13:12:45



diff --git a/OneDrive/Documentos/NetBeansProjects/Ca2/src/main/java/Calculator/Calculator.java b/OneDrive/Documentos/NetBeansProjects/Ca2/src/main/java/Calculator/Calculator.java
new file mode 100644
index 0000000..69d4178
--- /dev/null
+++ b/OneDrive/Documentos/NetBeansProjects/Ca2/src/main/java/Calculator/Calculator.java
@@ -0,0 +1,69 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Calculator;
+       import java.util.Scanner;
+       
+
+/**
+ *
+ * @author leanpadilla
+ */
+class Calculator {
+
+    private static Object MyCalc;
+    
+    public static void main(String[] args) {
+        
+       
+ int number1,number2;
+       String operation = "";
+       
+        Scanner input = new Scanner(System.in);
+        
+        System.out.println("Insert first number");
+        
+        number1 = input.nextInt();
+        
+        System.out.print("Select an operation +,-,*,/");
+        
+        operation = input.next();
+        
+        
+        System.out.println("Insert second number");
+        
+        number2 = input.nextInt();
+           
+        if (operation.equals("+")){
+        System.out.println("The sum result is:"+" "+(number1+number2));
+            
+    }
+        if (operation.equals("-")){
+        System.out.println("The subtraction result is:"+" "+(number1-number2));
+         
+  }
        if (operation.equals("*")){

            System.out.println("The multiplication result is:"+ " " + (number1 * number2))
        }
+ }
+   
+
+                                
+     
+        
+        
+        
+        
+        
+        
+        
+       
+                
+       
+       
+       
+       
+     
+
+       
+       
+       
+        
+        
+        
+        
+        
+        
+        
+        
+                
+             
+        
+        
+        
+        
+        
+        
+        
+        
+        
+        
+        
+                
+        
+        
+        
+                
+        
+    }
+    
+}
